### PR TITLE
Jobs page added.

### PR DESCRIPTION
**Pull request by uzairmirza90**

### CODE REVIEW

**Comment by getalihassan** [comment]
@uzairmirza90 resolve conflicts

**Comment by uzairmirza90** [comment]
> @uzairmirza90 resolve conflicts

Conflicts resolved.

**Comment by getalihassan** [comment]
@uzairmirza90 can you resolve conflicts again on this? :')

**Comment by uzairmirza90** [comment]
@getalihassan  Please merge https://github.com/prixite/prixite.github.io/pull/44 PR first, otherwise it will show conflicts again

File: components/JobCard/JobCard.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    image: string,
+    description: string,
+}
+
+
+const JobCard: React.FC<JobCardProps> = ({name, image, description}) => {
+  return (
+    <Container maxWidth="md">
+      <Stack
+        direction={{ xs: 'column', md: 'row' }}
```
Comment (getalihassan) [inline components/JobCard/JobCard.tsx:16]: avoid inline styling and try to use MUI styling hook

File: components/JobCard/JobCard.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        spacing={5}
+        my={3}
+        p={2}
+        sx={{
+          alignItems: 'center',
+          boxShadow: 'rgba(0, 0, 0, 0.24) 0px 3px 8px',
```
Comment (getalihassan) [inline components/JobCard/JobCard.tsx:22]: avoid inline styling and try to use MUI styling hook

File: components/JobCard/JobCard.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          <Image src={image} width={150} height={150} objectFit='cover' />
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: 1,
+          }}
+        >
+          <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
```
Comment (getalihassan) [inline components/JobCard/JobCard.tsx:36]: avoid inline styling and try to use MUI styling hook

File: components/JobCard/JobCard.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            <Typography fontSize={18} variant="h6">
```
Comment (getalihassan) [inline components/JobCard/JobCard.tsx:37]: avoid inline styling and try to use MUI styling hook

File: styles/pagesStyles/jobs.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
+.jobs-page {
```
Comment (getalihassan) [inline styles/pagesStyles/jobs.scss:1]: try to avoid scss if necessary

**Review by getalihassan — CHANGES_REQUESTED** [review]

**Review by getalihassan — APPROVED** [review]

**Comment by umairwaheed** [comment]
@uzairmirza90 can make the CI pass?

**Review by umairwaheed — CHANGES_REQUESTED** [review]
CI is not passing.

**Comment by uzairmirza90** [comment]
@umairwaheed  CI passed

**Review by umairwaheed — CHANGES_REQUESTED** [review]
@uzairmirza90 we don't include virtual environment in Git. Please remove venv folder.

**Review by umairwaheed — CHANGES_REQUESTED** [review]
@uzairmirza90 please do not delete yarn.lock file.

**Comment by uzairmirza90** [comment]
@umairwaheed  yarn.lock file added

**Review by umairwaheed — APPROVED** [review]
LGTM

**Pull request merged**